user-org-actor/src/main/java/org/sunbird/user/dao/IUserOSDao.java

```diff
@@ -12,6 +13,7 @@
  * @author Amit Kumar
  */
 public interface IUserOSDao {
```

Comment:
Use only one DAO (not separate for OS, ES ) - interface will be one and it implementation will be different.